sys/include/net/sock/dtls.h

```diff
@@ -678,7 +690,7 @@ ssize_t sock_dtls_recv_buf(sock_dtls_t *sock, sock_dtls_session_t *remote,
  * @return  -ENOMEM, if no memory was available to send @p data.
```

Comment:
Shouldn't this also add `-ETIMEOUT` for when `timeout` is hit?

Reply:
added `-ETIMEDOUT`